### PR TITLE
[Doc] Translate os/seL4.md to zh

### LINKED ISSUE #1077: [Docs] Translate seL4.md to zh
### The MD file link you want to translate (must be in English) 
https://github.com/WasmEdge/WasmEdge/blob/master/docs/book/en/src/os/sel4.md

### Target language
https://github.com/WasmEdge/WasmEdge/blob/master/docs/book/zh/src/os/sel4.md

### PR DESCRIPTION
**Pull request by bbcfive**
fix #1077 

Signed-off-by: Sijing Zhang bbcfive@163.com

### CODE REVIEW

**Comment by bbcfive** [comment]
@hydai @ibmibmibm @q82419 @alabulei1 can anyone help review this?

**Comment by alabulei1** [comment]
> @hydai @ibmibmibm @q82419 @alabulei1 can anyone help review this?

Sorry for my late reply. I will review it tonight. Thank you!

**Comment by alabulei1** [comment]
@bbcfive  Please check out the feedback. You also translated the image! Much appreciated.

**Comment by bbcfive** [comment]
@alabulei1 Thanks for the review! But I didn't see any feedback😂

**Comment by alabulei1** [comment]
> @alabulei1 Thanks for the review! But I didn't see any feedback😂

So wired! 

I paste my comments here, please check out. @bbcfive 

1.  you 翻译你
2. 第7行，WasmEdge Runtime 不用翻译
3. 第8行，在 seL4 后加一个空格
4. 第33行，结尾加个句号
5. 第43行，可以不加我们的
6. 第51行，all-in-one 翻译成一体化， all 就不用再翻译了，所以去掉所有。
7. 第79行，build.sh 前空一行，并且后面忘记加 `
8. 第141行， root 应该 `` , 前后记得空格哈

File: docs/book/zh/src/os/sel4.md [hunk 1]
```diff
@@ -1 +1,188 @@
...
+
+软件: 安装了开发工具包（例如：Python）的 Ubuntu 20.04。 我们推荐 [GitHub Actions Ubuntu 20.04 VM](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md) (查看 [已安装的 apt 包](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md#installed-apt-packages) 列表). 或者，你可以使用我们的 Docker 镜像 (查看 [Dockerfile](https://github.com/second-state/wasmedge-seL4/blob/main/docs/Dockerfile.sel4_build)).
+
+```bash
+$ docker pull wasmedge/sel4_build
+$ docker run --rm -v $(pwd):/app -it wasmedge/sel4_build
+(docker) root#
+```
+
+> 如果你不想自己搭建seL4系统模拟器，你可以从我们的 GitHub Actions 下载 [build artifact](https://github.com/second-state/wasmedge-seL4/actions/runs/1374510169), 并直接跳到 [启动 wasmedge-seL4](#boot-wasmedge-sel4)。
```
Comment (alabulei1) [inline docs/book/zh/src/os/sel4.md:33]: seL4 前后空一格
[启动 wasmedge-seL4](#boot-wasmedge-sel4) 这里应该是 #启动-wasmedge-sel4？

**Comment by alabulei1** [comment]
Also, please [set your DCO pass](https://www.secondstate.io/articles/dco/). So that we could merge your PR.

**Comment by bbcfive** [comment]
Signed-off-by: Sijing Zhang [bbcfive@163.com](mailto:bbcfive@163.com)

**Review by hydai — APPROVED** [review]

**Pull request merged**